source/uri-options/uri-options.rst

```diff
@@ -269,6 +277,14 @@ pertaining to URI options apply here.
      - no
      - Amount of time spent attempting to send or receive on a socket before timing out; note that this only applies to application operations, not SDAM
 
+   * - srvServiceName
+     - a valid SRV service name according to `RFC 6335 <https://datatracker.ietf.org/doc/html/rfc6335#section-5.1>`_; can be longer than 15 characters as long as the 63 (62 with prepended underscore)
+       character DNS query limit is not surpassed
+     - "mongodb"
+     - no
+     - the service name to use for SRV lookup in `initial DNS seedlist discovery <https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst>`_
+       and `SRV polling <https://github.com/mongodb/specifications/blob/master/source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst>`_
```

Comment:
Relative URIs should be used for references to other specs. e.g. `../source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst`